docs/src/enterprise/architecture.md

```diff
@@ -1,37 +1,30 @@
 ---
 title: Architecture
-description: lakeFS Enterprise architecture explained!!!!!1
+description: lakeFS Enterprise architecture explained!
 ---
 
 
 # Architecture
 
 
-The lakeFS Enterprise software consists of two components:
-1. lakeFS Enterprise: a proprietary version of lakeFS which is based on the OSS version and includes advanced functionality.
-2. A proprietary component called **Fluffy** which includes lakeFS' Enterprise features.
+lakeFS Enterprise extends the open-source lakeFS foundation, delivering a complete data versioning and governance solution with seamlessly integrated enterprise features like SSO, RBAC, mounting capabilities, and more.
 
-![img.png](../assets/img/enterprise/enterprise-arch.png)
+![img.png](../assets/img/enterprise/lakefs-enterprise-architecture.png)
 
 [1] Any user request to lakeFS via Browser or Programmatic access (SDK, HTTP
 API, lakectl).
 
-[2] Reverse Proxy (e.g. NGINX, Traefik, K8S Ingress): will handle user requests
-and proxy between lakeFS server and fluffy server based on the path prefix
-while maintaining the same host.
+[2] A reverse proxy (e.g., NGINX, Traefik, Kubernetes Ingress) will handle user requests between lakeFS server instances.
 
-[3] lakeFS server - the main lakeFS service.
+[3] lakeFS Enterprise - lakeFS with additional enterprise functionality, including advanced security, compliance, audit logging, and enterprise support.
 
-[4] fluffy server - service that is responsible for the Enterprise features.,
-it is separated by ports for security reasons.
+1. SSO authentication (Browser login via Azure AD, Okta, Auth0)
+1. RBAC authorization and policy enforcement
```

Comment:
Those 2 lines look out of context, like enumeration of something but very random. 
Perhaps add them to the above line i.e 
```
lakeFS Enterprise - lakeFS with additional enterprise functionality, including advanced security, SSO authentication, RBAC authorization, compliance, audit logging, and enterprise support.
```